schema_enforcer/schemas/manager.py

```diff
@@ -61,7 +61,13 @@ def create_schema_from_file(self, root, filename):  # pylint: disable=no-self-us
         # schema_type = "jsonschema"
         base_uri = f"file:{root}/"
         schema_full = jsonref.JsonRef.replace_refs(file_data, base_uri=base_uri, jsonschema=True, loader=load_file)
-        return JsonSchema(schema=schema_full, filename=filename, root=root)
+        schema = JsonSchema(schema=schema_full, filename=filename, root=root)
+        # Only add valid jsonschema files and print an error if an invalid file is found
+        valid = all((result.passed() for result in schema.check_if_valid()))
+        if not valid:
+            print(f"ERROR: File {filename} is not a valid jsonschema.")
```

Comment:
Please can you raise an exception here instead of printing directly to the console and terminating the process, ideally the exception should be handled in `cli.py`

Reply:
Also would be good to print (in cli.py) the error message(s) back to the console to provide some insight about why the file is not a valid jsonschema

Reply:
Good point on handling this in the CLI. I will refactor to use an exception.

Regarding the error messages, another option is to log an error that recommends running `schema-enforcer schema --check`. However, check appears to require unit tests and can't be run without them, so that would need to change.